thematters/matters-web
# feat(test): add article comment test

Test FE using Playwright [docs](https://matterslab.notion.site/FE-testing-Using-Playwright-f1572cdd74b04437bc04a47743adde75);
Looks like the test result(on FireFox) vary from local to Github CI/CD. 
### Test detail
1. How to test article publishing? Is it necessary to generate the exact article url and expect it?
- Login;
- Fill out title and content;
- Publish and view article; here we expect the article with the prefix like: "_https://web-develop.matters.news/@dev_testing/_"
2. How to test comment notice?
- Alice logins, goes to Bob's article and comments;
- Bob logins, checks notification, clicks the comment notice and sees the comment below;
3. How to test comment reply notice?
- Alice logins, goes to Bob's article, reply with "Reply to me!!!" comment with random generated comment;
- Bob logins, checks notification, finds the generated comment notice and be directed to the article page;
4. How to test article archive?
- Login;
- Publish article and view article;
- Click More Action and archive;
- Check if the "article has been archive" text ever exists on article page;
